cl2-component-library/src/components/LateralScrollControls/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import { useTheme } from 'styled-components';
+import Box from '../Box';
+import Button from '../Button';
+import testEnv from '../../utils/testUtils/testEnv';
+interface Props {
+  containerRef?: React.RefObject<HTMLDivElement>; // Ref of the container with elements which can be laterally scrolled through
+  scrollBtnDistanceMobile?: number; // Lateral scroll distance on arrow click for mobile
+  scrollBtnDistanceDesktop?: number; // Lateral scroll distance on arrow click for desktop
+  isSmallerThanPhone?: boolean;
+  children: React.ReactNode;
+}
+
+/*
+ * LateralScrollControls:
+ * Wraps a scrollable container with lateral scroll arrow buttons to scroll left and right.
+ */
+const LateralScrollControls = ({
```

Comment:
Do you think it makes more sense to name this `HorizontalScroll`? It's more clear in my head but it might be subjective.